### ISSUE TITLE
Possible options for args in operator deployment

### ISSUE THREAD

**Issue by gmichels** [Dynatrace/helm-charts#15]
Hello,

In https://github.com/Dynatrace/helm-charts/blob/fa4baebbf8912930eadcb6d09bcde3f79ea0ca1a/dynatrace-oneagent-operator/values.yaml#L24 one can set the OneAgent arguments such as enable log access.

I am interested in setting the host group, and in the manual deployment, OneAgent has arguments in this form:

```
    - name: dynatrace-oneagent
      image: 'docker.io/dynatrace/oneagent:latest'
      args:
        - APP_LOG_CONTENT_ACCESS=1
        - HOST_GROUP=GROUP_NAME
```

What are the available options in the `args` list?

Thanks

**Comment by lrgar** [comment]
Hi there,

First, args and some other parameters aren't customizable on the latest released Helm chart (v0.6.0) but will be on the next, soon to be released, v0.7.0.

Once that is available, the field is an array, so it would be something like,

```
helm install --set '.....,oneagent.args={APP_LOG_CONTENT_ACCESS=1, HOST_GROUP=GROUP_NAME}'
```

See Helm documentation for details, https://helm.sh/docs/intro/using_helm/

**Comment by gmichels** [comment]
Does master reflect 0.6.0? Seems like it is customizable now:

https://github.com/Dynatrace/helm-charts/blob/fa4baebbf8912930eadcb6d09bcde3f79ea0ca1a/dynatrace-oneagent-operator/templates/Common/customresource.yaml#L27

**Comment by lrgar** [comment]
The master branch represents our latest changes, so what would be v0.7.0

As for v0.6.0, it looks like this,

https://github.com/Dynatrace/helm-charts/blob/292548ae59283b5a60c1eb2f8e22e7ce79023bd9/dynatrace-oneagent-operator/values.yaml#L15-L28

**Comment by lrgar** [comment]
This is now available on the Helm Chart v0.7.0
